src/iconsManifest/supportedExtensions.ts

```diff
@@ -2584,6 +2584,11 @@ export const extensions: IFileCollection = {
       extensions: ['juice'],
       format: FileFormat.svg,
     },
+    {
+      icon: 'cmrc',
+      extensions: ['cmrc.json'],
```

Comment:
This won't work as a file extension. VS Code only takes the latest extension. It doesn't work with composed extensions.

Reply:
i got inspired from some files already existed, 

i want to decorate the whole cmrc.json file. any suggestion how to ?

Reply:
actualy `.cmrc` as file name is better than  `.cmrc.json` 
I updated the [extension config](https://github.com/Ar-mane/component-maker/releases)
with new commit, it works as it suppose to